src/__test__/__mocks__/home.test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.